src/attachment_loader.rs

```diff
@@ -4,20 +4,12 @@ use crate::{
         spAttachmentLoader_dispose,
     },
     c_interface::{NewFromPtr, SyncPtr},
-    Atlas, Attachment, AttachmentType, RegionProps, Skin,
+    Atlas, Attachment, AttachmentType, RegionProps, Skin, SpineError,
 };
 
-/// Error types related to [`AttachmentLoader`](`crate::AttachmentLoader`).
-#[derive(Debug)]
-pub enum AttachmentLoaderError {
-    /// Creating an attachment failed.
-    /// Check [`error1`](`Self::error1`) and [`error2`](`Self::error2`) for more information.
-    CreateAttachmentFailed,
-    InvalidArgument {
-        field: &'static str,
-    },
-}
-
```

Comment:
I think we shouldn't introduce a new error type. It causes people to juggle two types. I consolidated into the existing [`SpineError`].